managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedger.java

```diff
@@ -323,14 +321,16 @@
     public boolean isTerminated();
 
     /**
-     * Returns managed-ledger config 
```

Comment:
unrelated change, adds noise to an already large review.

Reply:
Thanks, this is the auto changing of intellij-idea.